docs/using-radicle/fetching-and-merging.md

```diff
@@ -1,133 +1,63 @@
-```
-
-or for those of you who prefer rebasing:
-
-```
-$ git rebase merle@hyb1exienq5txww75rgdftuf6j78qehtigma6hn19zt8gz5pu5k73g/main
-[main 3e0e32d] Radicle Update
- 1 file changed, 2 insertions(+)
-Successfully rebased and updated refs/heads/main.
-```
-
-This puts your working copy in a state this up-to-date with your contributor's
-changes. Once you have commited changes to your working copy, you can publish
-them changes to the Radicle network by running `git push rad`(See [Pushing
-changes][pc]):
-
-```
-$ git push rad main
-Enumerating objects: 2, done.
-Counting objects: 100% (2/2), done.
-Delta compression using up to 8 threads
-Compressing objects: 100% (2/2), done.
-Writing objects: 100% (2/2), 1.72 KiB | 1.72 MiB/s, done.
-Total 2 (delta 0), reused 0 (delta 0), pack-reused 0
-To rad://hwd1yre8wtxufyqnt8xzxt8ps3n7ww8qzpfmud5pibarumshawnpcrjzjme.git
-   e6ee76d..57917bf  main -> main
-```
-If you are connected to the network, these changes will also be published to the
-network and gossiped to your connected peers. If you are a maintainer, this
-means that anybody who tracks your project will be able to find, view, and fetch
-your updates to the project.
-
-Make sure that your client is running when you run `git push rad` to ensure
-changes are successfully published. If you can't see your changes in Upstream,
-see the [FAQ][fq] for more help.
-
-
-[pc]: using-radicle/pushing-changes.md
+## Viewing patches
+
+Now that you're tracking the contributor, you can navigate to the patches tab on your project and see any patches that contributor has created.
+
+<!-- TODO: replace with final image -->
+
+![Patches][pa]
+
+You can then click on the patch that you are interested in and get a detail view. You'll see the title of the patch, the description, which branches it's comparing and the commits. If you're the maintainer of the project, you'll also see two buttons, one to checkout the contribution and another to merge it.
+
+<!-- TODO: replace with final image -->
+
+![PatchDetail][pd]
+
+## Checking out & merging a patch
+
+If you're the maintainer of a project, you can easily checkout a patch by clicking the checkout button and copying the command.
+
+`git fetch rad tags/radicle-patch/<patch-name>:tags/radicle-patch/<peer-name>/<patch-name>`
+
+<!-- TODO: replace with final image -->
+
+![PatchDetailCheckout][pdc]
+
+If you've like the patch, you can then chose to merge it by copying the command that's in the modal when you click the merge button.
+<!-- TODD: replace with actual merge command -->
+
+`git fetch rad tags/radicle-patch/<patch-name>:tags/radicle-patch/<peer-name>/<patch-name>`
+
+<!-- TODO: replace with final image -->
+
+![PatchDetailMerge][pdm]
+
+That should merge the patch into your main branch, if you then push the changes `git push rad` you'll have officially accepted the patch. When a patch has the same history as main, it will automatically show up as closed.
```

Comment:
Proposal:

That command merges the patch into your main branch. When you push the changes with `git push rad` you'll have accepted and merged the patch. […]